### PR TITLE
Give more time for flaky test

### PR DESCRIPTION
**Pull request by joshklop**

### CODE REVIEW

**Review by omerfirmak — APPROVED** [review]
I am not sure if it will help, since there is a retry loop already.

**Comment by joshklop** [comment]
> I am not sure if it will help, since there is a retry loop already.

Good point. The retry loop is a little buggy though: we end up doing the test four times no matter what (which could lead to us failing on the fourth try). I'll remove the loop now that we're waiting longer.

**Comment by joshklop** [comment]
When I remove the timeout altogether, the test sometimes stalls for 10 minutes (before it is timed out by the go tool itself). This PR clearly isn't a solution; I'll need to investigate further.

**Comment by omerfirmak** [comment]
Originally timeout was longer, I lowered it when I added the retry loop. Lets fix the loop and see how it goes?

File: p2p/p2p_test.go [hunk 1]
```diff
@@ -76,15 +76,20 @@ func TestService(t *testing.T) {
 		require.NoError(t, err)
 		t.Cleanup(closer)
 
-		for i := 0; i < 4; i++ {
+		maxRetries := 4
+	RetryLoop:
```
Comment (omerfirmak) [inline p2p/p2p_test.go:80]: do we need this label?

**Pull request merged**